sphinx/testing/fixtures.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+"""
+    sphinx.testing.fixtures
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    Sphinx test fixtures for pytest
+
+    :copyright: Copyright 2007-2017 by the Sphinx team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+from __future__ import print_function
+
+import os
+import subprocess
+import sys
+from collections import namedtuple
+from tempfile import gettempdir
+
+import pytest
+from six import StringIO, string_types
+
+from . import util
+
+if False:
+    from typing import Dict, Union  # NOQA
+
+
+@pytest.fixture(scope='session')
+def rootdir():
+    return None
```

Comment:
rootdir is not provided by `sphinx.testing` because it depends test implementation.
In Sphinx itself, we define rootdir for Sphinx in `tests/conftest.py`